doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -500,13 +508,30 @@ Interface Physical Property Models
 Surface Tension Models
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Lethe supports two types of surface tension models: ``constant`` and ``linear``. A ``constant`` surface tension model assumes a constant value of surface tension, while a ``linear`` surface tension assumes that the surface tension evolves linearly with the temperature:
+Lethe supports three types of surface tension models: ``constant``, ``linear``, and ``phase change``. A ``constant`` surface tension model assumes a constant value of surface tension, while a ``linear`` surface tension assumes that the surface tension evolves linearly with the temperature:
 
 .. math::
   \sigma(T) = \sigma_0 + \frac{d\sigma}{dT} (T-T_0)
 
 where :math:`\sigma_0` is the ``surface tension coefficient`` evaluated at ``reference state temperature`` :math:`T_0` and :math:`\frac{d\sigma}{dT}` is the ``surface tension gradient`` with respect to the temperature :math:`T`.
 
+The ``phase change`` model also assumes linear variation of the surface tension, however it considers the liquid fraction :math:`\alpha_\mathrm{l}` to apply the associated force only if the fluid is liquid:
```

Comment:
I am not sure I understand what you want to say here

Reply:
I would try to rewrite it a bit more directly. If T<T_solidus, surface tension is zero, otherwise the linear model applies. Something like that..